Resources/PageRegistration/registration.json

```diff
@@ -32,25 +32,16 @@
       }
     ],
 	"AjaxRequests": [
-	{
-		"Name": "Ajax",
-		"IsHealthy": true,
-		"AjaxEndpoint": "https://localhost:44306/api/get/find-a-course/search/{0}/ajax",
-		"HealthCheckRequired": true,
-		"OfflineHtml": "<div class=\"govuk-width-container\"><H3>Help Service Unavailable</H3></div>",
-		"DateOfRegistration": "2020-10-22T12:49:24.9672393Z",
-		"LastModifiedDate": "2020-10-26T14:16:10.4948477Z"
-	},
-	{
-		"Name": "IsValidPostcode",
-		"IsHealthy": true,
-		"AjaxEndpoint": "https://localhost:44306/api/get/find-a-course/search/{0}/isvalidpostcode",
-		"HealthCheckRequired": true,
-		"OfflineHtml": "<div class=\"govuk-width-container\"><H3>Help Service Unavailable</H3></div>",
-		"DateOfRegistration": "2020-10-22T12:49:24.9672393Z",
-		"LastModifiedDate": "2020-10-26T14:16:10.4948477Z"
-	}
-    ],
+    {
+      "Name": "Ajax",
+      "IsHealthy": true,
+      "AjaxEndpoint": "https://localhost:44306/api/get/find-a-course/search/{0}/ajax",
```

Comment:
Can you change this URL to:

https://__AppServiceName__.__appServiceDomain__/api/get/find-a-course/search/{0}/ajax